paper/paper.md

```diff
@@ -1,351 +1,412 @@
+In addition to the standard case of an assumed constant surface tension of water, three thermodynamic frameworks describing the surface-partitioning of organic species have been included in PySDM. These models describe the surface tension of a droplet as a function of the dry aerosol composition and the wet radius. An example of how to specify the surface tension formulation is shown below. The three additional thermodynamic frameworks have been implemented following @Ovadnevaite_et_al_2017, @Ruehl_et_al_2016, and Szyszkowski-Langmuir. The `some_aerosol` object is an instance of an arbitrary aerosol from the `DryAerosolMixture` super-class.
+```python
+formulae_bulk = Formulae(
+    surface_tension='Constant'
+)
+formulae_ovad = Formulae(
+    surface_tension='CompressedFilmOvadnevaite',
+    constants={
+        'sgm_org': 35 * si.mN / si.m,
+        'delta_min': 1.75 * si.nm
+    }
+)
+formulae_ruehl = Formulae(
+    surface_tension='CompressedFilmRuehl',
+    constants={
+        'RUEHL_nu_org': some_aerosol.modes[0]['nu_org'],
+        'RUEHL_A0': 115e-20 * si.m * si.m,
+        'RUEHL_C0': 6e-7,
+        'RUEHL_m_sigma': 0.3e17 * si.J / si.m**2,
+        'RUEHL_sgm_min': 35 * si.mN / si.m
+    }
+)
+formulae_sl = Formulae(
+    surface_tension='SzyszkowskiLangmuir',
+    constants={
+        'RUEHL_nu_org': some_aerosol.modes[0]['nu_org'],
+        'RUEHL_A0': 115e-20 * si.m * si.m,
+        'RUEHL_C0': 6e-7,
+        'RUEHL_sgm_min': 35 * si.mN / si.m
+    }
+)
+```
 
+Using these different models for the surface-partitioning, we can demonstrate the effect variable surface tension has on the activation of aerosol with some organic fraction. The presence of the orgnaics both modifies the surface tension and the hygroscopicity, resulting sometimes in a Köhler curve with local minima features. Below is (psuedo-)code used to generate four Köhler curves for the same partially organic aerosol particle, just under different assumptions of surface-partitioning by the insoluble organic species. 
 ```python
-from PySDM.physics.constants_defaults import rho_w
-from matplotlib import pyplot
-
-for step in [0, 1200, 2400, 3600]:
-    particulator.run(step - particulator.n_steps)
-    pyplot.step(
-        x=radius_bins_edges[:-1] / si.um,
-        y=particulator.products['dv/dlnr'].get().squeeze() * rho_w/si.g,
-        where='post', label=f"t = {step}s")
-
-pyplot.xscale('log')
-pyplot.xlabel(r'particle radius [$\mu$ m]')
-pyplot.ylabel("dm/dlnr [g/m$^3$/(unit dr/r)]")
-pyplot.legend()
-pyplot.show()
+model = formulae.surface_tension.__name__
+sigma = formulae.surface_tension.sigma(T, v_wet, v_dry, some_aerosol.modes[0]['f_org'])
+RH_eq = formulae.hygroscopicity.RH_eq(r_wet, T, some_aerosol.modes[0]['kappa'][model], r_dry**3, sigma)
+plot(r_wet, (RH_eq - 1)*100)
 ```
+![Köhler curves for aerosol under 4 assumptions of thermodynamic surface-partitioning of organic species.](surf_fig_kohler.pdf){#fig:kohler width="60%"}
```

Comment:
@slayoo This "example" is not yet merged into main. It's something that will be part of the paper Ryan and I working on, but that paper won't be ready in time for this JOSS submission. Should I try to merge a first version of this example that just has the Köhler curves now?